src/Hl7.Fhir.Base/Serialization/BaseFhirJsonPocoDeserializer.cs

```diff
@@ -510,6 +524,12 @@ FhirJsonPocoDeserializerState state
             // to simply create a list by Adding(). Not the fastest approach :-(
             int elementIndex = 0;
             bool? onlyNulls = null;
+            elementIndex = delayedValidations.ParsingPropertyValue(propertyName);
+            if (elementIndex > 0)
+            {
+                state.Path.IncrementIndex(elementIndex);
+                state.Errors.Add(ERR.DUPLICATE_ARRAY(ref reader, state.Path.GetInstancePath()));
```

Comment:
Add duplicate array to the list of recoverable errors.

Reply:
I think this is already added